APE_LTS.rst

```diff
@@ -0,0 +1,109 @@
+significant API changes from version to version due to the youth of the project and long term
+support releases were a way to shield against these changes, or at least reduce disruption to
+be every two years.
+
+LTS releases are however not free:
+
+* While we use infrastructure to automate backporting fixes to release branches,
+  in practice the LTS branch diverges more and more from the main development
+  branch and automated backports are more likely to fail as time goes on,
+  requiring human effort to manually do the backports, which can sometimes be
+  non-trivial.
+
+* Even once fixes are backported to the LTS branch, carrying out releases is not
+  fully automated and requires manual steps, so having to do
+  bugfix releases for both the latest release branch and LTS branch puts an extra
+  burden on the core package release team.
+
+In addition, it is not clear that LTS releases are needed anymore - now that the
+core package is over ten years old, it has reached a much higher level of
+stability. No other main package in the scientific Python ecosystem has long
+term support releases, and there is no longer a strong argument for having this
+in astropy either.
+
+This APE proposes the following:
+
+* No longer designate any releases as LTS
+* Keep the 6-month release cycle and bump releases to x.0 every two years
+* Refer to the x.0 releases every two years (e.g. v6.0) as 'major'
+  releases, other releases every 6 months (e.g. v6.1) as 'minor', and bug fixing
+  releases (e.g. v6.1.2) as 'bugfix releases'
+* Minimize as much as possible any changes that break API or change results in
+  minor releases, but being pragmatic and accepting such changes if they would
+  not affect many users
+* Add deprecation warnings or future warnings in minor and major releases but only
+  go through with the breaking changes in major releases (either the next major
+  release following when a warning was added, or subsequent ones if the changes
+  are deemed to be high impact enough to wait longer).
+
+With this in place, we would effectively be striving to maximize stability over
+two year periods but users would see deprecation warnings to prepare them for
+changes. This is in contrast to the current LTS system where users might not see
+a deprecation warning until the next LTS release as deprecations are not typically
+backported.
+
+
+Branches and pull requests
+--------------------------
+
+N/A
+
+Implementation
+--------------
+
+This APE proposes to continue supporting the 5.0.x LTS series but to not
+designate 6.0 as LTS.
+
+
+Backward compatibility
+----------------------
+
+N/A
+
+
+Alternatives
+------------
+
+If there were any alternative solutions to solving the same problem, they should
+be discussed here, along with a justification for the chosen approach.
+
+
+Decision rationale
+------------------
+
+<To be filled in by the coordinating committee when the APE is accepted or rejected>
```

Comment:
Being the coauthor and current CoCo member, I would recuse myself from CoCo discussions regarding this APE.